### ISSUE TITLE
`scrollEdgeAppearance="transparent"` keep the tabLabel color gray no matter if selected or not (iOS)(expo-router)

### ISSUE THREAD

**Issue by guillaumehcht** [okwasniewski/react-native-bottom-tabs#164]
### Before submitting a new issue

- [X] I tested using the latest version of the library, as the bug might be already fixed.
- [X] I tested using a [supported version](https://github.com/reactwg/react-native-releases/blob/main/docs/support.md) of react native.
- [X] I checked for possible duplicate issues, with possible answers.

### Bug summary

Whenever I add the `scrollEdgeAppearance="transparent"` to my Tabs, the text below the icon in the TabBar doesn't take the same color as the icon anymore.

**What I tried to fix it but didn't worked**:
- Removed/Added `tabBarActiveTintColor`
- Removed/Added `sidebarAdaptable`
- Removed/Added `translucent`
- Removed/Added `tabLabelStyle`

I have truly no idea what's causing this, I was using the 0.6.0 and I updated to the 0.7.0 this morning and still experiencing the same issue.

Note, I am using:
- RN0.76
- Expo SDK52
- Expo-Router
- react-native-bottom-tabs 0.7.0

**I am adding 2 videos for context, one with `scrollEdgeAppearance="transparent"` and one without.**

https://github.com/user-attachments/assets/3de26cb8-4b48-41f3-8fe3-f794fa4e6311

https://github.com/user-attachments/assets/6704ab6f-1574-4df7-a1a4-af62c2f44cd8

### Library version

0.7.0

### Environment info

```shell
"dependencies": {
    "@bottom-tabs/react-navigation": "^0.7.0",
    "@expo-google-fonts/public-sans": "^0.2.3",
    "@expo/vector-icons": "^14.0.2",
    "@gorhom/bottom-sheet": "^5.0.6",
    "@invertase/react-native-apple-authentication": "^2.4.0",
    "@react-native-async-storage/async-storage": "1.23.1",
    "@react-native-community/datetimepicker": "8.2.0",
    "@react-native-firebase/app": "^21.4.0",
    "@react-native-firebase/auth": "^21.4.0",
    "@react-native-firebase/firestore": "^21.4.0",
    "@react-native-firebase/storage": "^21.4.0",
    "@react-native-google-signin/google-signin": "^13.1.0",
    "@react-native-menu/menu": "^1.1.6",
    "@react-navigation/bottom-tabs": "^7.0.0",
    "@react-navigation/native": "^7.0.0",
    "@tanstack/eslint-plugin-query": "^5.60.1",
    "@tanstack/react-query": "^5.59.20",
    "@types/ngeohash": "^0.6.8",
    "blurhash": "^2.0.5",
    "burnt": "^0.12.2",
    "color": "^4.2.3",
    "date-fns": "^4.1.0",
    "eslint": "^8.57.0",
    "expo": "~52.0.11",
    "expo-apple-authentication": "~7.1.2",
    "expo-application": "~6.0.1",
    "expo-blur": "~14.0.1",
    "expo-build-properties": "~0.13.1",
    "expo-constants": "~17.0.3",
    "expo-dev-client": "~5.0.4",
    "expo-font": "~13.0.1",
    "expo-haptics": "^14.0.0",
    "expo-image": "~2.0.2",
    "expo-image-picker": "~16.0.3",
    "expo-linear-gradient": "~14.0.1",
    "expo-linking": "~7.0.3",
    "expo-localization": "~16.0.0",
    "expo-location": "~18.0.2",
    "expo-notifications": "~0.29.8",
    "expo-router": "~4.0.9",
    "expo-splash-screen": "~0.29.13",
    "expo-status-bar": "~2.0.0",
    "expo-symbols": "~0.2.0",
    "expo-system-ui": "~4.0.4",
    "expo-updates": "~0.26.9",
    "expo-web-browser": "~14.0.1",
    "i18next": "^23.16.5",
    "intl-pluralrules": "^2.0.1",
    "ngeohash": "^0.6.3",
    "posthog-react-native": "^3.3.10",
    "pressto": "^0.2.0",
    "react": "18.3.1",
    "react-dom": "18.3.1",
    "react-i18next": "^15.1.1",
    "react-native": "0.76.3",
    "react-native-bottom-tabs": "^0.7.0",
    "react-native-gesture-handler": "~2.20.2",
    "react-native-get-random-values": "~1.11.0",
    "react-native-gifted-chat": "^2.6.4",
    "react-native-ios-context-menu": "^2.5.2",
    "react-native-ios-utilities": "^4.5.1",
    "react-native-keyboard-controller": "^1.14.4",
    "react-native-keychain": "^9.2.1",
    "react-native-reanimated": "~3.16.1",
    "react-native-safe-area-context": "4.12.0",
    "react-native-screens": "^4.0.0",
    "react-native-svg": "15.8.0",
    "react-native-unistyles": "^2.12.0",
    "react-native-web": "~0.19.13",
    "react-native-webview": "13.12.2",
    "zeego": "^2.0.4",
    "zustand": "^5.0.1"
  },
```

### Steps to reproduce

1. Create a new project using `npx create-expo-app@latest`
2. Install `react-native-bottom-tabs` and `@bottom-tabs/react-navigation`
3. Create the `BottomTabs.tsx` component
4. Create a `(Tabs)` folder with 2 tabs at least
5. Setup your `_layout.tsx`
6. Add and Remove `scrollEdgeAppearance='transparent'` to see the issue (sometimes you need to hit reload to see the change FYI)

### Reproducible sample code

```js
import React from 'react';
import { Tabs } from '@/components/BottomTabs';
import { Platform } from 'react-native';

const TabLayout = () => {
	return (
		<Tabs scrollEdgeAppearance='transparent'>
			<Tabs.Screen
				name='index'
				options={{
					title: 'Home',
					tabBarIcon: () =>
						Platform.select({
							ios: { sfSymbol: 'bell.fill' },
							android: require('@/assets/icons/notifications.svg'),
						}),
				}}
			/>
			<Tabs.Screen
				name='explore'
				options={{
					title: 'Explore',
					tabBarIcon: () =>
						Platform.select({
							ios: { sfSymbol: 'message.fill' },
							android: require('@/assets/icons/chat.svg'),
						}),
				}}
			/>
		</Tabs>
	);
};

export default TabLayout;
```

**Comment by okwasniewski** [comment]
Hey, thanks for opening the issue. 

Apple has a weird way of handling this natively where `scrollEdgeAppearance="transparent"` requires totally different config, this is probably an oversight. I will look into this

**Comment by okwasniewski** [comment]
@guillaumehcht Should be fixed in 0.7.1 🎉
